### PR TITLE
Update FileUploadNotificationReceiver to forward the SSLContext info

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public synchronized void open() throws IOException
         // Codes_SRS_SERVICE_SDK_JAVA_AMQPFILEUPLOADNOTIFICATIONRECEIVE_25_003: [The function shall create an AmqpsReceiveHandler object to handle reactor events]
         if (amqpReceiveHandler == null)
         {
-            amqpReceiveHandler = new AmqpFileUploadNotificationReceivedHandler(this.hostName, this.userName, this.sasToken, this.iotHubServiceClientProtocol, this, this.proxyOptions);
+            amqpReceiveHandler = new AmqpFileUploadNotificationReceivedHandler(this.hostName, this.userName, this.sasToken, this.iotHubServiceClientProtocol, this, this.proxyOptions, this.sslContext);
```
Comment (abhipsaMisra) [inline service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java:88]: @timtay-microsoft What would be the best test class to test this?

**Review by vinagesh — APPROVED** [review]

**Review by barustum — APPROVED** [review]

File: service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java (hunk 1 above)
Comment (timtay-microsoft) [reply service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFileUploadNotificationReceive.java:88]: Unit tests, or in the e2e tests that we have that actually use the file upload notification receiver. I'm pretty sure the e2e tests are all disabled though

**Review by timtay-microsoft — APPROVED** [review]
Thanks for catching this!

**Pull request merged**